app/controllers/password_credentials_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PasswordCredentialsController < ApplicationController
+  require_unauthenticated_access
+  before_action :ensure_manual_login_allowed
+
+  layout "public"
+
+  def edit
+    @user = find_signed_user(params[:token])
+  end
+
+  def update
+    user = find_signed_user(params[:token])
+
+    if user.password_credential&.update(update_params)
```

Comment:
ah yeah I ran into that when I was testing early on actually. That's a good point. They wouldn't actually get to this point because I check in the job before I send the email, but maybe I shouldn't do that and send a token even if they don't have a password credential. I'll update it to create if it doesn't exist. I could definitely seeing this being confusing for users they way it currently is